src/tool_fsharp/RunFsc.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+set -e
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ "$SOURCE" != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
```

Comment:
Something to consider longer team is that maybe we want these helper scripts to be written in pure sh, so the shebang can go back to `/bin/sh` and folks don't have to have the bash shell installed to actually use dotnet.  I know we require the bash shell to be around to build our individual repositories (and it seems totally reasonable to say that and depend on bashism's there) but maybe we can get pure `sh` scripts at some point (or the need goes away, I sort of remember some discussion somewhere about giving MSBuild more intimate knowledge about the fact that it's running under a host and letting it use that information when running core apps)